alwaysontop/render.js

```diff
@@ -234,19 +263,38 @@ class AlwaysOnTop extends EventEmitter {
         // cross-origin redirect can cause any set global variables to be blown
         // away.
         this._alwaysOnTopWindow = window.open('', 'AlwaysOnTop');
-        if(!this._alwaysOnTopWindow) {
+        if (!this._alwaysOnTopWindow) {
             return;
         }
         this._alwaysOnTopWindow.alwaysOnTop = {
             api: this._api,
-            onload: this._updateLargeVideoSrc,
+            onload: () => {
+                this._updateLargeVideoSrc();
+                const size = this._alwaysOnTopBrowserWindow.getSize();
+                previousWidth = size[0];
+                previousHeight = size[1];
+            },
             onbeforeunload: () => {
                 this.emit(ALWAYSONTOP_WILL_CLOSE);
                 this._api.removeListener(
                     'largeVideoChanged',
                     this._updateLargeVideoSrc
                 );
             },
+            onresize: debounce(() => {
```

Comment:
Maybe for Mac you can use https://electronjs.org/docs/api/browser-window#winsetaspectratioaspectratio-extrasize-macos

Reply:
I'm working on windows so it's a bit tricky to test it out. But with the changes I made it should work just fine with the current logic on MAC as well